packages/react-core/src/components/NotificationBadge/NotificationBadge.tsx

```diff
@@ -12,22 +12,24 @@ export enum NotificationBadgeVariant {
 }
 
 export interface NotificationBadgeProps extends Omit<ButtonProps, 'variant'> {
-  /** @deprecated Use the variant prop instead - Adds styling to the notification badge to indicate it has been read */
-  isRead?: boolean;
-  /** Determines the variant of the notification badge */
-  variant?: NotificationBadgeVariant | 'read' | 'unread' | 'attention';
-  /** A number displayed in the badge alongside the icon */
-  count?: number;
-  /** content rendered inside the notification badge */
-  children?: React.ReactNode;
-  /** additional classes added to the notification badge */
-  className?: string;
-  /** Adds accessible text to the notification badge. */
+  /** Adds an accessible label to the notification badge. */
   'aria-label'?: string;
-  /** Icon to display for attention variant */
+  /** Icon to display for attention variant. */
   attentionIcon?: React.ReactNode;
-  /** Icon do display in notification badge */
+  /** Content rendered inside the notification badge. */
+  children?: React.ReactNode;
+  /** Additional classes added to the notification badge. */
+  className?: string;
+  /** A number displayed in the badge alongside the icon. */
+  count?: number;
+  /** Icon to display in the notification badge. */
   icon?: React.ReactNode;
+  /** Flag for indicating whether the notification badge is expanded. */
```

Comment:
I would update this comment to say it applies expanded styling to the badge.  The badge is not actually expanded here.  It was a little unclear to me at first to what the prop was doing.